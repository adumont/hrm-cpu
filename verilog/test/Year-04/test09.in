16 a9
16 2a
41 13
20 bc
19 dc
37 97
40 26
42 c9
01 32
15 d3
32 fb
54 fe
16 1d
26 5d
17 3c
54 60
03 b1
18 47
36 9b
48 40
57 e7
23 9c
04 4c
21 b6
06 ba
43 6b
56 f8
18 dc
41 81
20 a1
03 f6
15 cc
41 52
28 71
49 0d
