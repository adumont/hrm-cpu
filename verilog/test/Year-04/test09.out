2a
a9
bc
13
97
dc
c9
26
d3
32
fe
fb
5d
1d
60
3c
47
b1
40
9b
9c
e7
b6
4c
6b
ba
dc
f8
a1
81
cc
f6
71
52
